### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by kmaric69**
I'm submitting the third assignment.

### CODE REVIEW

File: 03.CSS_class_manipulation/src/cssClassManipulation.js [hunk 1]
```diff
@@ -1,5 +1,18 @@
-var cssClass = (function() {
-  'use strict';
-
-  // code goes here
-})();
\ No newline at end of file
+'use strict';
+var cssClass = {
+  add: function(htmlElement, className){
```
Comment (badurinantun) [inline 03.CSS_class_manipulation/src/cssClassManipulation.js]: Add space after `)` when writing functions

File: 03.CSS_class_manipulation/src/cssClassManipulation.js [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+    const selectedElement = htmlElement.classList.add(className);
```
Comment (badurinantun) [inline 03.CSS_class_manipulation/src/cssClassManipulation.js]: What's the value of `selectedElement`, are you expected to return a value here?

File: 03.CSS_class_manipulation/src/cssClassManipulation.js (hunk 1 above)
Comment (kmaric69) [reply 03.CSS_class_manipulation/src/cssClassManipulation.js]: 👍

File: 03.CSS_class_manipulation/src/cssClassManipulation.js (hunk 2 above)
Comment (kmaric69) [reply 03.CSS_class_manipulation/src/cssClassManipulation.js]: It's undefined. There is no need to define it as a variable, nor to return it.

File: 03.CSS_class_manipulation/src/cssClassManipulation.js [hunk 3]
```diff
@@ -1,5 +1,15 @@
-var cssClass = (function() {
-  'use strict';
-
-  // code goes here
-})();
\ No newline at end of file
+'use strict';
+var cssClass = {
```
Comment (badurinantun) [inline 03.CSS_class_manipulation/src/cssClassManipulation.js]: Use `const` instead

**Pull request merged**